editor/editor_settings.cpp

```diff
@@ -216,17 +216,83 @@ Variant _EDITOR_DEF(const String &p_var, const Variant &p_default) {
 	return p_default;
 }
 
+static Dictionary _get_builtin_script_templates() {
+	Dictionary templates;
+
+	//No Comments
+	templates["no_comments.gd"] =
+			"extends %BASE%\n"
+			"\n"
+			"func _ready():\n"
+			"%TS%pass\n";
+
+	//Empty
+	templates["empty.gd"] =
+			"extends %BASE%"
+			"\n"
+			"\n";
+
+	//Default Extended
+	templates["default_extended.gd"] =
+			"extends %BASE%\n"
+			"# ctrl+click or cmd+click to get information about the Node type this script is attached to.\n"
+			"# any variable and function is available in the scope of this script.\n"
+			"# don't forget to also checkout the classes this node Inherits from! All those functions + parameters are available too.\n"
+			"\n"
+			"\n"
+			"func _ready():\n"
+			"	# if you dont need one of the callbacks set it to false: set_process(false)"
+			"\n"
+			"	set_process(true) 		# This tells godot to call the '_ready' callback  on this node each frame\n"
```

Comment:
`set_process*` is not necessary in 3.0. It will be enabled at the presence of _process function (and the likes).
The function can be useful for disabling & enabling it at runtime. Might be better to let the users know about this somehow.

(also double spaces after "callback")